skyportal/tests/frontend/test_groups.py

```diff
@@ -32,9 +32,9 @@ def test_add_new_group(driver, super_admin_user, user):
     driver.refresh()
     driver.get('/groups')
     driver.wait_for_xpath('//input[@name="name"]').send_keys(test_proj_name)
-    driver.wait_for_xpath('//input[@name="groupAdmins"]').send_keys(user.username)
-    driver.save_screenshot('/tmp/screenshot1.png')
-    driver.click_xpath('//button[contains(.,"Create Group")]')
+    driver.click_xpath('//div[@id="groupAdminsSelect"]')
+    driver.wait_for_xpath_to_be_clickable(f'//li[text()="{user.username}"]').click()
```

Comment:
```suggestion
    driver.wait_for_xpath_to_be_clickable(
        f'//li[contains(text(),"{user.username}")]'
    ).click()
```
The ripple span in the li element messes up the text()= xpath matching I think. Using the contains seems to work for me locally.

Reply:
Interesting, thanks for looking into that! I couldn't reproduce the errors locally. Much appreciated.

Reply:
@acrellin Just kidding I might be wrong about that. I think I looked at the wrong test and thought this fixed made it pass.